Tests/QueueInteropTransportTest.php

```diff
@@ -385,7 +389,9 @@ public function testSendWithMessageMetadata()
     public function testSendWithBadMessageMetadata()
     {
         $this->expectException(MissingMessageMetadataSetterException::class);
-        $this->expectExceptionMessageRegExp('/Missing "setDumb" setter for "dumb" metadata key in "Double\\\Enqueue\\\MessengerAdapter\\\Tests\\\Fixtures\\\DecoratedPsrMessage\\\[^"]+" class/');
+        $this->expectErrorMessage(
+            'Missing "setDumb" setter for "dumb" metadata key in "Double\Enqueue\MessengerAdapter\Tests\Fixtures\DecoratedPsrMessage\P18" class'
```

Comment:
Inject just `::class` here

Reply:
Could you please clarify what you mean

Reply:
Is it a link to a file in another project?

Reply:
I just deleted it because it's not so important to spend time on it.
DecoratedPsrMessage::class gets us 'Enqueue\MessengerAdapter\Tests\Fixtures\DecoratedPsrMessage' but I need 'Double\Enqueue\MessengerAdapter\Tests\Fixtures\DecoratedPsrMessage\P18' and looks like "P18" might be chnaged